_pages/ato/types.md

```diff
@@ -23,7 +23,7 @@ You may operate without authorization if all of the following conditions are met
         * IP CIDR block
         * some kind of auth mechanism
             * HTTP Basic Auth
-            * OAuth (MyUSA)
+            * OAuth ([Cloud.gov Auth](https://docs.cloud.gov/apps/leveraging-authentication/))
```

Comment:
Minor nit: mind making "cloud.gov" lower-case? (per our [content guide](https://github.com/18F/cg-product/blob/master/ContentGuide.md#use-these-exact-strings-for-tricky-product-names-abbreviations-and-team-names) Also, I'd add add GitHub auth as an option.